spec/rails_helper.rb

```diff
@@ -12,7 +12,7 @@
 require 'factory_bot_rails'
 
 FactoryBot.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
-FactoryBot.find_definitions
+# FactoryBot.find_definitions
```

Comment:
Had to comment out this line, otherwise it kept complaining that the Address factory was already defined.

Reply:
weird.  let me know if you think we should just kill the gem, move models/migrations and services into core app...

Reply:
i'm not sure if that's related to the gem, but maybe?